src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```
@@ -15,7 +15,7 @@
     <PackageId>NuGet.CommandLine</PackageId>
     <TargetFramework>$(NETFXTargetFramework)</TargetFramework>
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
-    <Description>NuGet Command Line Tool.</Description>
+    <Description>NuGet Command Line Tool</Description>
```

```suggestion
    <Description>NuGet Command Line Interface</Description>
```

Our docs refer to nuget.exe as "nuget.exe CLI"